Source/Diagnostics/Diagnostics.cpp

```diff
@@ -72,6 +72,31 @@ Diagnostics::BaseReadParameters ()
         }
     }
 
+    // default for writing field output is 1
+    int write_fields = 1;
+    pp.query("write_fields", write_fields);
+    if (write_fields == 0) {
+        if (m_format == "checkpoint") {
+            ablastr::warn_manager::WMRecordWarning(
+                "Diagnostics",
+                "For checkpoint format, write_fields flag must be 1. Overwriting."
+            );
+            pp.add("write_fields", 1);
+        }
+        else if (m_format == "plotfile") {
```

Comment:
Technically, we can probably make this: if plotfile AND lab diagnostics.

For BTD, this works. We can unify the flags in a follow-up PR.